### ISSUE TITLE
Blinking line while creating new edge

### ISSUE THREAD

**Issue by mawilk** [cytoscape/cytoscape.js-edgehandles#24]
When you try to add an edge the line you drag from source node constantly blinks. You can even find position where line becomes invisible. When you comment lines 290-292 - that is:

if( now - lastDrawLine < lineDrawRate ){
    return;
}

line stops to blink and is drawn properly. Is this some kind of optimalisation?

**Comment by maxkfranz** [comment]
Yes, it's an optimisation.  I'll see if I can reproduce the flickering you mentioned.
